src/main/java/Deadline.java

```diff
@@ -0,0 +1,28 @@
+public class Deadline extends Task {
+    String date;
+
+    Deadline(String name, boolean isDone, String date) {

```

Comment:
I don't think the param for isDone is necessary as when tasks are generated, `isDone` will always be `false`.